spec/unit/models/services/service_instance_spec.rb

```diff
@@ -183,6 +183,31 @@ module VCAP::CloudController
         expect(ServiceInstanceAnnotationModel.where(id: annotation.id)).to be_empty
       end
 
+      it 'cascade deletes all ServiceInstanceOperations for this instance' do
```

Comment:
In theory there should only be a single operation - and this test also instantiates a single operation, thus I would change the text to `it 'cascade deletes the related ServiceInstanceOperation for this instance'`.

Reply:
ok